.github/workflows/molecule-actions-core-repositories_server.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: core/repositories_server unit testing
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'roles/core/repositories_server/**'
+      - '.github/workflows/molecule-actions-core-repositories_server.yml'
+  pull_request:
+    paths:
+      - 'roles/core/repositories_server/**'
+      - '.github/workflows/molecule-actions-core-repositories_server.yml'
+
+jobs:
+  molecule_test:
+    name: ${{ matrix.distro }} - molecule test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+          - centos:7
+          - centos:8
+
+    env:
+      MOLECULE_DISTRO: ${{ matrix.distro }}
+      PY_COLORS: 1
+      ANSIBLE_FORCE_COLOR: 1
+    steps:
+      - name: Set INSTANCE_DISTRO variable
+        run: echo "::set-env name=INSTANCE_DISTRO::${MOLECULE_DISTRO/:/}"
```

Comment:
I don't see any usage of INSTANCE_DISTRO in the test. Do you need this env variable?